Dockerfile

```diff
@@ -1,16 +1,17 @@
 # Dockerfile
-FROM node:lts
+FROM node:16.13.1-alpine3.14
```

Comment:
lts version isn't fine?

Reply:
When you run yarn install locally, KodaDot requires very specific node version. I forgot which package it is, but I would like to stick to specific versions instead of just always using LTS.  

More often than not, packages don't update at the same speed as node does. It makes no sense to have the build failed randomly one day because the version changed.

Reply:
Okay!